salt/pillar/vmware_pillar.py

```diff
@@ -329,10 +329,10 @@ def ext_pillar(minion_id, pillar, **kwargs):  # pylint: disable=W0613
     if "property_types" in kwargs:
         for prop_type in kwargs["property_types"]:
             if isinstance(prop_type, dict):
-                property_types.append(getattr(vim, prop_type.keys()[0]))
-                if isinstance(prop_type[prop_type.keys()[0]], list):
+                property_types.append(getattr(vim, next(iter(prop_type.keys()))))
```

Comment:
The same value is calculated 3 times at once. Please put it to a variable.

Reply:
done